sdks/go/pkg/beam/model/PROTOBUF.md

```diff
@@ -24,44 +24,38 @@ If you make changes to .proto files, you will need to rebuild the generated Go c
 First, follow this one-time setup:
 
 1. Download [the protobuf compiler](https://github.com/google/protobuf/releases).
-   The simplest approach is to download one of the prebuilt binaries and extract
-   it somewhere in your machine's `$PATH`.
-1. A properly installed Go development environment per [the official
-   instructions](https://golang.org/doc/install). `$GOPATH` must be set properly.
-   If it's not set, follow
-   [these instructions](https://github.com/golang/go/wiki/SettingGOPATH).
+   The simplest approach is to download one of the prebuilt binaries (named
+   `protoc`) and extract it somewhere in your machine's `$PATH`.
 1. Add `$GOBIN` to your `$PATH`. (Note: If `$GOBIN` is not set, add `$GOPATH/bin`
    instead.)
 
 To generate the code:
 
 1. Navigate to this directory (`pkg/beam/model`).
-1. `go get -u github.com/golang/protobuf/protoc-gen-go`
+1. `go install google.golang.org/protobuf/cmd/protoc-gen-go@latest`
+1. `go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest`
```

Comment:
Instead of recommending @latest, we should have the instructions ensure the go.mod version is being used which is the default behavior as of go 1.16:

https://golang.org/ref/mod#go-install

IIUC this means adding a required version to our go.mod in the chance we aren't already depending on a specific version.

That would avoid version random increments, and we can manage the updates more intentionally, and hopefully, get folks to do this more automatically when they change the protos. 

Especially since the instructions below for failures end up saying "either move to the go mod version, or update the go mod version".

Reply:
Done. This also let me shorten the instructions by a lot.